lib/services/auth.dart

```diff
@@ -109,8 +109,8 @@ class AuthenticationService {
         .signInWithOAuth(Provider.facebook, redirectTo: LOGIN_REDIRECT_URL);
   }
 
-  Future signInWithApple() async {
-    await _client.auth.signInWithApple();
+  Future<AuthResponse> signInWithApple() async {
+    return _client.auth.signInWithApple();
```

Comment:
Apple don't provide the full name after login to `Supbase` we need to get it manually from the response